helm/examples/dev/templates/ingress.yaml

```diff
@@ -1,11 +1,13 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: aleph-ingress-dev
-  namespace: dev
+  name: aleph-ingress
+  namespace: default
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
```

Comment:
Perhaps we can fix this warning:

```
annotation "kubernetes.io/ingress.class" is deprecated, please use 'spec.ingressClassName' instead
```

Reply:
That happens when you blindly follow StackOverflow answers. Thanks for catching it, this is now fixed!